snapcraft.yaml

```diff
@@ -0,0 +1,24 @@
+# based on https://github.com/snapcore/snapcraft/blob/master/demos/gradle/snap/snapcraft.yaml
+
+name: jabref
+version: '4.0-dev'
```

Comment:
This value needs to be documented (that it has to be changed) if it cannot be parametrized.